crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1011,201 +1007,135 @@ where
         match result {
             Ok(result) => Ok(result),
             Err(state) => {
-                debug!(
-                    ?tx_digest,
-                    num_errors = state.errors.iter().map(|e| e.1.len()).sum::<usize>(),
-                    num_unique_errors = state.errors.len(),
-                    good_stake = state.good_stake,
-                    bad_stake = state.bad_stake,
-                    num_signatures = state.signatures.len(),
-                    "Received signatures response from validators handle_transaction"
-                );
-                if !state.errors.is_empty() {
-                    debug!(?tx_digest, "Errors received: {:?}", state.errors);
-                }
-
+                self.record_process_transaction_metrics(tx_digest, &state);
                 let state = Self::record_non_quorum_effects_maybe(tx_digest, state);
                 Err(QuorumSignTransactionError {
                     total_stake: self.committee.total_votes,
-                    good_stake: state.good_stake,
+                    good_stake: state.good_stake(),
                     errors: state.errors,
                     conflicting_tx_digests: state.conflicting_tx_digests,
                 })
             }
         }
     }
 
-    fn handle_response_with_err(
+    fn record_process_transaction_metrics(
+        &self,
+        tx_digest: &TransactionDigest,
+        state: &ProcessTransactionState,
+    ) {
+        let num_signatures = state.tx_signatures.validator_sig_count();
+        self.metrics.num_signatures.observe(num_signatures as f64);
+        let good_stake = state.good_stake();
+        self.metrics.num_good_stake.observe(good_stake as f64);
+        self.metrics.num_bad_stake.observe(state.bad_stake as f64);
```

Comment:
I'm not sure why these metrics are useful to export to a dashboard, we never looked at them